core/templates/domain/summary/learner-exploration-summary.model.ts

```diff
@@ -51,7 +51,7 @@ export interface LearnerExplorationSummaryBackendDict {
 
 export class LearnerExplorationSummary {
   constructor(
-      public category: string,
+      public category: string | undefined,
```

Comment:
Where exactly can `category` be `undefined`?

Reply:
![image](https://user-images.githubusercontent.com/68547101/96962063-ad45bf00-1523-11eb-830b-4120d72952c1.png)
The method createFromBackendDict returns a new dict of type LearnerExplorationSummary with category to be undefined.

Reply:
This gave the following error -
![image](https://user-images.githubusercontent.com/68547101/96962410-5987a580-1524-11eb-8150-8634f45e34c0.png)

Reply:
Typescript returns undefined when it doesn't get a value. However,Assigning a value initially is giving linting error. Can you please help me with this one?

Reply:
@vojtechjelinek can you please help in this one?

Reply:
Why the category can be `undefined`? It seems to me that it is just being passed from `dict` to the `LearnerExplorationSummary`.